apps/heft/src/cli/actions/CustomAction.ts

```diff
@@ -138,12 +96,6 @@ export class CustomAction<TParameters> extends HeftActionBase {
           getParameterValue = () => parameter.values;
           break;
         }
-
-        default: {
-          throw new Error(
```

Comment:
This was removed for the same reason as above (the value in default is `never` because all valid cases are handled). So whatever we decide on the above case, I'll apply here too.